### DISCUSSION TITLE
Are the training datasets publicly available?

### DISCUSSION THREAD

**Discussion by Majdoddin** [openai/whisper#430] [Q&A]
Thank you for amazing and open source Whisper 👍 

As I understand it from the paper, you collected the proper training data from internet, and then did some preprocessing on it (re-sampling, ...). _Is this training data publicly available?_ 
I need it in the process of saprsifying the whisper's model using [ `sparseeml`](https://github.com/neuralmagic/sparseml). Of course I could use standard datasets that are used for evaluation of whisper, but whisper's robustness is based on using a diverse training dataset.

**Comment by JaejinCho** [comment]
I had the same question, too although the collected data does not seem open to the public.

**Comment by phineas-pta** [comment]
unfortunately no, even the training code aren't public (they only provide fine tune code)
